src/components/home/elements/IssueToken.vue

```diff
@@ -0,0 +1,760 @@
+<template>
+  <b-modal :id="'issueTokenModal_' + tokenId"
+           centered
+           lazy
+           title="IssueToken"
+           hide-footer
+           hide-header
+           ref="issueTokenModal"
+           :busy="true"
+           @hidden="resetPage">
+    <button
+      class="close btn-close"
+      @click="endSend">
+      <img src="../../../assets/imgs/icons/operate/ic_close.svg">
+    </button>
+    <b-tabs @input="hideQrScan">
+      <b-tab title="Hot Wallet"
+             :disabled="walletType === 'coldWallet'"
+             :active="walletType==='hotWallet'">
+        <b-container
+          class="text-left"
+          v-if="pageId===1">
+          <b-form-group label="Wallet Address"
+                        label-for="address-input">
+            <b-form-input id=address-input
+                          class="address-input"
+                          readonly
+                          v-model="address"
+                          :state="isValidIssuer(address)"
+                          aria-describedby="inputLiveFeedback"></b-form-input>
+            <b-form-invalid-feedback id="inputLiveFeedback">
+              Cannot issue token. You are not issuer of this token.
+            </b-form-invalid-feedback>
+            <b-btn
+              block
+              variant="light"
+              disabled
+              class="balance-input"
+              readonly>
+              <span class="balance-title">
+                <img src="../../../assets/imgs/icons/wallet/ic_token2.svg"
+                     width="20"
+                     height="20">
+              </span>
+              <span class="balance">Issue Available{{ formatter(maxSupply - currentSupply) }}</span>
+            </b-btn>
+          </b-form-group>
+          <b-form-group label="Issue Amount"
+                        label-for="amount-input">
+            <b-form-input id="amount-input"
+                          class="amount-input"
+                          v-model="amount"
+                          aria-describedby="inputLiveFeedback"
+                          :state="isAmountValid('hot')">
+            </b-form-input>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-if="isWrongFormat(amount)">
+              Invalid format. The number of digits after the decimal point may be larger than the token precision.
+            </b-form-invalid-feedback>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else-if="isTokenInsufficient(amount)">
+              Insufficient token
```

Comment:
weird to use `Insufficient token` and fun name `isTokenInsufficient`  here, since the situation is issue amount > max - current